<launch>
    <!-- Bring in global hardware parameters -->
    <rosparam command="load" file="$(find system_globals)/params/frames.yaml" />
    <rosparam command="load" file="$(find system_globals)/params/hardware_constants.yaml" />
    <rosparam command="load" file="$(find topic_messages)/params/topics.yaml" />
    <rosparam command="load" file="$(find walking_gait)/params/gait_state.yaml" />

    <!-- Needed for max servo rate -->
    <rosparam command="load" file="$(find system_globals)/params/servos.yaml" />

    <node
        pkg="walking_gait"
        name="leg_ikine_node"
        type="leg_ikine_node.py"
        respawn="true"
        respawn_delay="1"
        output="screen"
    />
</launch>